core/src/main/scala/quasar/physical/mongodb/fs/queryfile.scala

```
@@ -161,9 +161,9 @@ private final class QueryFileInterpreter[C](
       }).liftM[QRT].liftM[WorkflowExecErrT]
 
     case FileExists(file) =>
-      collFromPathM(file)
-        .flatMap(MongoDbIO.collectionExists(_).liftM[FileSystemErrT])
-        .run.liftM[QRT].liftM[WorkflowExecErrT]
+      Collection.fromPathy(file).fold(
```

I am still somewhat concerned with this behavior, i.e. simply returning `false` when there is something wrong with the path being provided. Either way, I think it's probably good that it's not part of the core algebra but it might make sense for the mongodb interpreter to report this as a failure in the monad stack rather than simply returning `false`. But we also already discussed this, so I can get on board with just returning `false`.